src/frontend/src/components/Wallet.vue

```diff
@@ -20,13 +20,23 @@ import { mapState } from "vuex";
       (newValue: any, oldValue: any) => {
         if (newValue.length > oldValue.length) {
           this.$store.dispatch("ACTION_FETCH_WALLET_AMOUNT");
-
-          // Displays toast if transaction was recieved
-          const message = "New transaction received!";
-          this.$store.dispatch("ACTION_DISPLAY_TOAST", {
-            message: message,
-            type: "success",
-          });
+          const toSearch = []
+          let i: number;
```

Comment:
alternatively, you can do `newValue.find( (value, index) = { if (index > oldValue.length) .... })`

Reply:
correction `index > oldValue.length - 1`

Reply:
Updated, please check new solution